extensions/amp-story/1.0/amp-story.css

```diff
@@ -183,13 +183,24 @@ amp-story amp-sidebar[side][open] {
   max-width: 280px !important;
   min-width: 280px !important;
   width: 280px !important;
+  transform: translate3d(0, 0, 0) !important;
+  transition: transform 0.2s cubic-bezier(0.4, 0.0, 1, 1) !important;
   display: block !important;
 }
 
 [desktop] amp-sidebar[side][open]{
   max-width: 320px !important;
 }
 
+amp-story amp-sidebar[side] {
+  max-width: 280px !important;
+  min-width: 280px !important;
+  width: 280px !important;
+  transform: translate3d(100%, 0, 0) !important;
+  transition: transform 0.3s cubic-bezier(0.0, 0.0, 0.2, 1) !important;
+  display: block !important;
```

Comment:
Now that these rules (max-width, width...) are set here, could we remove them from the other `amp-story amp-sidebar[side][open] {}` block?

Reply:
Definitely. Done.